test/TestUtils/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compile": {
+      "include": [
+        "../../build_projects/shared-build-targets-utils/Utils/DotNetCli.cs",
+        "../../build_projects/shared-build-targets-utils/Utils/EnvVars.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/AnsiConsole.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/AnsiColorExtensions.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/ArgumentEscaper.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildReporter.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/CommandResult.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Constants.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Reporter.cs"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-004338"
```

Comment:
We run into an issue here in that the Microsoft.NETCore.App we are generating has a higher version that that of what is available in the SDK

---
In reply to: [66107545](https://github.com/dotnet/core-setup/pull/97#discussion_r66107545) [](ancestors = 66107545)